### PR TITLE
Add CSE pass description in doxygen documentation

### PR DESCRIPTION
**Pull request by gaurides**

### CODE REVIEW

**Comment by yimeisun123** [comment]
1) The CI failed on the clang format check. Please run the apply format script to fix it. 
2) Add the comment on how to include additional op in CSE.

File: src/ngraph/pass/cse.hpp [hunk 1]
```diff
@@ -26,6 +26,17 @@ namespace ngraph
     }
 }
 
+/// The Common Subexpression Elimination pass removes duplicate computations in a given computation graph.\n
+/// Two commutations are considered to be duplicate to each other if both applies same operation on same set of inputs,
```
Comment (yimeisun123) [inline src/ngraph/pass/cse.hpp:30]: 1. Please change to "computations".
2. both "apply"

Comment (gaurides) [reply src/ngraph/pass/cse.hpp]: Done.

**Comment by gaurides** [comment]
> 1. The CI failed on the clang format check. Please run the apply format script to fix it.
> 2. Add the comment on how to include additional op in CSE.

Done. Please do check the description in cse.cpp on how to add CSE for new op

**Comment by gaurides** [comment]
This is how the output looks like
image

**Comment by gaurides** [comment]
TEST NOW

**Comment by raramer01** [comment]
TESTNOW_DLDT_E2E

**Pull request merged**